### PR TITLE
Fix the link to the GCE getting-started guide

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

**Comment by mikebrow** [comment]
though does not need testing as it's a link only and we don't have link tests :)

File: docs/kube-up.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ tar -C ${CRI_CONTAINERD_PATH} -xzf cri-containerd-${VERSION}.linux-amd64.tar.gz
 . ${CRI_CONTAINERD_PATH}/opt/containerd/cluster/gce/env
 ```
 ## Create Kubernetes Cluster on GCE
-Follow these instructions [here](https://kubernetes.io/docs/getting-started-guides/gce/) to create a production quality Kubernetes cluster on GCE.
+Follow these instructions [here](https://kubernetes.io/docs/setup/turnkey/gce/) to create a production quality Kubernetes cluster on GCE.
```
Comment (Random-Liu) [inline docs/kube-up.md]: Can you also fix the `kube-up.sh` link above?

**Comment by Random-Liu** [comment]
I'll get this merged and send another PR to fix the link above.

**Pull request merged**